### PR TITLE
New template: dell_powerconnect

### PR DESCRIPTION
**Pull request by nidebr**
Hi,

ISSUE TYPE
New Template Pull Request

COMPONENT
- dell_powerconnect_show_interfaces_description
- dell_powerconnect_show_interfaces_status
- dell_powerconnect_show_bridge_address_table

SUMMARY
add some templates for dell_powerconnect

BR
Nick

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
Hmm, weird travis issues, can you make a benign commit to kick off travis again? Normally as an admin I could kick it off, but I can't seem to get it to work in this case.

**Comment by nidebr** [comment]
Hi, @itdependsnetworks.
Travis is OK now ;)
BR
Nick

File: templates/dell_powerconnect_show_interfaces_description.textfsm [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+Value DESCRIPTION (\S.*?)
+
+Start
+  ^Port\s+Description -> Begin
+
+Begin
+  ^-+
+  ^Ch\s+Description -> End
+  ^${INTERFACE}(?:\s+${DESCRIPTION})?\s*$$ -> Record
+
```
Comment (jmcgill298) [inline templates/dell_powerconnect_show_interfaces_description.textfsm:11]:
```suggestion
  ^\s*
  ^. -> Error
```

File: templates/dell_powerconnect_show_interfaces_description.textfsm [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Value INTERFACE (\S+)
+Value DESCRIPTION (\S.*?)
```
Comment (jmcgill298) [inline templates/dell_powerconnect_show_interfaces_description.textfsm:2]: Does using `(\S*)` work? then that will simplify the regex match below?

File: templates/dell_powerconnect_show_interfaces_description.textfsm [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+Start
+  ^Port\s+Description -> Begin
+
+Begin
+  ^-+
+  ^Ch\s+Description -> End
+  ^${INTERFACE}(?:\s+${DESCRIPTION})?\s*$$ -> Record
```
Comment (jmcgill298) [inline templates/dell_powerconnect_show_interfaces_description.textfsm:10]: in combination with the above comment, can this just be `^${INTERFACE}\s*${DESCRIPTION}\s*$$ -> Record`

File: templates/dell_powerconnect_show_interfaces_status.textfsm [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+Start
+  ^Port\s+Type\s+Duplex\s+Speed\s+Neg\s+ctrl\s+State\s+Pressure\s+Mode -> Begin
+
+Begin
+  ^\s+Flow\s+Link\s+Back\s+Mdix
+  ^-+
+  ^${PORT}\s+${TYPE}\s+${DUPLEX}\s+${SPEED}\s+${NEG}\s+${FLOWCTRL}\s+${LINKSTATE}\s+${BACKPRESSURE}\s+${MDIXMODE} -> Record
+  ^\s+Flow\s+Link
+  ^Ch\s+Type\s+Duplex\s+Speed\s+Neg\s+control\s+State -> End
```
Comment (jmcgill298) [inline templates/dell_powerconnect_show_interfaces_status.textfsm:19]:
```suggestion
  ^Ch\s+Type\s+Duplex\s+Speed\s+Neg\s+control\s+State -> End
  ^\s*
  ^. -> Error
```

**Review by jmcgill298 — CHANGES_REQUESTED** [review]
Thanks @nidebr 

Two of the templates need to add catching for unmatched lines, and I had a question about one match line.

File: templates/dell_powerconnect_show_interfaces_description.textfsm (hunk 2 above)
Comment (nidebr) [reply templates/dell_powerconnect_show_interfaces_description.textfsm:2]: Hi @jmcgill298, Yes that work, but your suggest is more simply.

File: templates/dell_powerconnect_show_interfaces_description.textfsm (hunk 3 above)
Comment (nidebr) [reply templates/dell_powerconnect_show_interfaces_description.textfsm:10]: Same comment above ;)

**Comment by jmcgill298** [comment]
Thanks @nidebr another PR was merged that caused a conflict, so I have resolved that and should be good to merge once travis completes

**Review by jmcgill298 — APPROVED** [review]

**Pull request merged**